Dockerfile

```diff
@@ -51,6 +51,8 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --allow-
     hipblas  \
     hipify-clang \
     half \
+    libtbb2 \
+    libtbb-dev \
```

Comment:
This is already handled in the install_preqs.sh script.

Reply:
Removed